File: IdenticalPairs.py
class Solution:
#1512 
    def numIdenticalPairs(self, nums: List[int]) -> int:
        totalIdenticalPairs = 0
        currentIndex = 0

        for x in nums:
            ind = currentIndex + 1
            while(ind < len(nums)):
                if(x == nums[ind]):
                    totalIdenticalPairs+=1
                ind+=1
            currentIndex+=1
        return totalIdenticalPairs
        

File: IdenticalPairs.java
class Solution 
{//1512
    public int numIdenticalPairs(int[] nums) 
    {
        int identicalPairAmount = 0;
        
        for(int i = 0; i < nums.length; i++)
        {
            
            for(int j = i + 1; j < nums.length; j++)
            {

                if(nums[i] == nums[j])
                    identicalPairAmount++;


            }

        }

        return identicalPairAmount;
        
    }
}